sqlmesh/__init__.py

```diff
@@ -1,3 +1,4 @@
+# ruff: noqa: E402
```

Comment:
I used this because we need `extend_sqlglot()` to happen right after it's imported to kick off the dialect patching. Otherwise we'd get a [module-import-not-at-top-of-file](https://docs.astral.sh/ruff/rules/module-import-not-at-top-of-file/#module-import-not-at-top-of-file-e402) error for the subsequent imports.